app/_includes/plugins/redis-cloud-auth.md

```diff
@@ -0,0 +1,170 @@
+* `$AWS_ACCESS_KEY_ID`: (Optional) Your AWS access key ID. 
+* `$AWS_ACCESS_SECRET_KEY`: (Optional) Your AWS secret access key.
+{% endnavtab %}
+{% navtab "Azure instance" %}
+
+You need a running Redis instance on an [Azure Managed Redis instance](https://learn.microsoft.com/en-us/azure/redis/entra-for-authentication) with Entra authentication configured.
+
+```yaml
+config:
+  storage: redis
+  storage_config:
+    redis:
+      host: ${instance_address}
+      username: ${instance_username}
+      port: 6379
+    cloud_authentication:
+      auth_provider: azure
+      azure_client_id: ${azure_client_id}
+      azure_client_secret: ${azure_client_secret}
+      azure_tenant_id: ${azure_tenant_id}
+```
+Replace the following with your actual values:
+* `$INSTANCE_ADDRESS`: The Azure Managed Redis instance address.
+* `$INSTANCE_USERNAME`: The Azure Managed Redis instance principal ID with essential access.
+* `$AZURE_CLIENT_ID`: Your Azure Managed Redis instance client ID.
+* `$AZURE_CLIENT_SECRET`: (Optional) Your Azure Managed Redis instance client secret. 
+* `$AZURE_TENANT_ID`: (Optional) Your Azure Managed Redis instance tenant ID.
+
+{% endnavtab %}
+{% navtab "Azure cluster" %}
+
+You need a running Redis instance on an [Azure Managed Redis cluster](https://learn.microsoft.com/en-us/azure/redis/entra-for-authentication) with Entra authentication configured.
+
+```yaml
+config:
+  storage: redis
+  storage_config:
+    redis:
+      cluster_nodes: ${cluster_address}
+      username: ${cluster_username}
+      port: 6379
+    cloud_authentication:
+      auth_provider: azure
+      azure_client_id: ${azure_client_id}
+      azure_client_secret: ${azure_client_secret}
+      azure_tenant_id: ${azure_tenant_id}
+```
+Replace the following with your actual values:
+* `$CLUSTER_ADDRESS`: The Azure Managed Redis cluster address.
+* `$CLUSTER_USERNAME`: The Azure Managed Redis cluster principal ID with essential access.
+* `$AZURE_CLIENT_ID`: Your Azure Managed Redis cluster client ID.
+* `$AZURE_CLIENT_SECRET`: (Optional) Your Azure Managed Redis cluster client secret. 
+* `$AZURE_TENANT_ID`: (Optional) Your Azure Managed Redis cluster tenant ID.
+
+{% endnavtab %}
+{% navtab "GCP instance" %}
+
+You need a running Redis instance on an [Google Cloud Memorystore instance](https://cloud.google.com/memorystore/docs/cluster/about-iam-auth).
+
+```yaml
+config:
+  storage: redis
+  storage_config:
+    redis:
+      host: ${instance_address}
+      port: 6379
+    cloud_authentication:
+      auth_provider: gcp
+      gcp_service_account_json: ${service_account}
+```
+Replace the following with your actual values:
+* `$INSTANCE_ADDRESS`: The Memorystore instance address.
+* `$GCP_SERVICE_ACCOUNT`: The GCP service account JSON.
+
+instance_address:
+    value: $INSTANCE_ADDRESS
+    description: The Memorystore instance address.
+  service_account:
+    value: $GCP_SERVICE_ACCOUNT
+    description: The GCP service account JSON.
```

Comment:
```suggestion
```

It seems this part is redundant.

<img width="1720" height="1198" alt="image" src="https://github.com/user-attachments/assets/080c9492-3740-4301-9440-1040ab561754" />